Makefile

```diff
@@ -1,17 +1,24 @@
 NGINX_PLUS_VERSION=15-2
 NGINX_IMAGE=nginxplus:$(NGINX_PLUS_VERSION)
 
-test: docker-build run-nginx-plus test-run clean
-	
+test: docker-build run-nginx-plus test-run config-no-stream test-no-stream clean
```

Comment:
I like it this way, so that we can change the config and `make test-run` or `make test-no-stream` without changing the state of the container.

I also don't find this confusing.